src/set_theory/game/impartial.lean

```diff
@@ -152,7 +152,7 @@ lemma le_zero_iff {G : pgame} [G.impartial] : G ≤ 0 ↔ 0 ≤ G :=
 by rw [le_zero_iff_zero_le_neg, le_congr (equiv_refl 0) (neg_equiv_self G)]
 
 lemma lt_zero_iff {G : pgame} [G.impartial] : G < 0 ↔ 0 < G :=
-by rw [lt_iff_neg_gt, neg_zero, lt_congr (equiv_refl 0) (neg_equiv_self G)]
+by rw [lt_iff_neg_gt, pgame.neg_zero, lt_congr (equiv_refl 0) (neg_equiv_self G)]
```

Comment:
why does `pgame` have its own `neg_zero`? do neither `neg_zero` nor `neg_zero'` work for it?

Reply:
No, `pgame` is pretty badly behaved. Most results about `pgame` are not equalities but double inequalities (`a ≤ b ∧ b ≤ a` instead of `a = b`). Once you take the `antisymmetrization`, you get `game`, which is much saner.